apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -466,6 +466,33 @@ export const getMultipleChoiceSectionSummary = (state) => {
   return results;
 };
 
+/**
+ * Selector function that takes in the state and a boolean indicating
+ * if the current assessment is a survey.
+ * Returns an integer total count of the number of students who have submitted
+ * the current assessment.
+ */
+export const getTotalSubmissionForCurrentAssessment = (state, isSurvey) => {
```

Comment:
Should `Submissions` be plural in this name?  I'd also consider naming this `countSubmissionsForCurrentAssessment` which strongly implies that the return value is an integer and hints that some nontrivial computation is going on when you call this method, not just a simple lookup in our state tree.